### ISSUE TITLE
Better solutions than enchantments.

### ISSUE THREAD

**Issue by TheOnlySilverClaw** [Guerra24/Voxel#69]
As the title says, since I think enchantments are a really uninspired way of upgrading weapons and armour (and enchantment tables in a smithy look ridiculous), I want to suggest replacements for most of them:
## General
- Unbreaking: Refined steel.
- Mending: Maybe some metal-magician spell?
## Armour
- Protection: Thicker armour, various spells.
- Fire protection: Leather armour, pyromancer armour.
- Feather falling : Do we really need that? I'm not sure....uhm...jetpacks? Trying not to fall down? A rope?
- Blast protection : Heavier, lined armour.
- Projectile Protection: Solid armour, e.g. plate instead of chain-mail.
- Respiration: A diving bell?
- Aqua affinity: hmmm....freeze the water?
- Thorns: Obviously, put some spikes on armour.
- Depth Strider: A boat! Better, ships!
- Frost Walker: Cryomancers do that anyway.
## Sword
- Sharpness: Use a whetstone to actually sharpen weapons.
- Smite: Holy weapons like a [Holy Cross](http://minecraft.curseforge.com/projects/reforged-cross-grave/images)? Light magic/blessings.
- Bane of Arthropods: Not that useful, since there is more animal/monster variety than in Minecraft. Instead, use fire if you really hate spiders.
- Knockback: Blunt weapons like maces.
- Fire Aspect: Fight with a torch or be a pyromancer.
- Looting: Use better tools to loot corpses, for example a skinning knife to get more leather.
## Bow
- Power: Use better arrows, bows or a crossbow instead.
- Punch: Use heavier arrows, bow with more drawing power or a crossbow instead.
- Flame: Craft flammable arrows (with oil, steel wool, etc.) or use pyromancy.
- Infinity: Use a boomerang instead?
## Fishing Rod
- Luck of the Sea: Implement a better system for getting treasures then by fishing!
- Lure: Use insects or pieces of meat as actual lure.
## Tools
- Efficiency: Use explosives to clear away large areas of stone.
- Silk Touch: Unnecessary, in my opinion.
- Fortune: Use better furnaces to smelt ore.

To summarize it, instead of using enchantments, I vote for a more advanced metallurgy system, where it is possible to craft weapons and armour with better properties and a wider variety of weapons and armour. Maybe even make advanced metallurgy+mineralogy+mechanics a magical domain or something similar.
Also, some enchantments can be replaced by more sophisticated magic, while others would become obsolete by changing related mechanics.
